pyserum/open_orders_account.py

```diff
@@ -84,9 +85,12 @@ def find_for_market_and_owner(
                 bytes=str(owner),
             ),
         ]
-
         resp = conn.get_program_accounts(
-            program_id, encoding="base64", memcmp_opts=filters, data_size=OPEN_ORDERS_LAYOUT.sizeof()
+            program_id,
+            commitment=Recent,
```

Comment:
We should allow an optional `commitment` parameter so the user can specify this.